torch/utils/_config_module.py

```diff
@@ -102,6 +102,10 @@ def __init__(
             assert isinstance(
                 self.default, bool
             ), f"justknobs only support booleans, {self.default} is not a boolean"
+            if self.value_type is not None:
```

Comment:
We want to check if env_name_default or env_name_force is set - Not value_type.

i.e.

Config(env_name_default="EXAMPLE_ENV_VAR", default=600) should throw an exception.

Reply:
Fixed in new version.